t/test-lib-functions.sh

```diff
@@ -754,6 +754,25 @@ projfs_start () {
 	trap projfs_stop EXIT
 }
 
+# Run the given command twice in parallel, wait for both to complete, and
```

Comment:
My only request in this review is actually a minor and nonfunctional change here -- would you mind shifting this function down a bit, so the `projfs_start()` and `projfs_stop()` functions still live together, just for the sake of clarity in the future?

Reply:
Of course! Done 👍